lib/highline/import.rb

```diff
@@ -1,3 +1,6 @@
+#!/usr/bin/env ruby
```

Comment:
Thanks for noticing it.
Do want to fix yourself so the commits go with your name?
If you prefer I can fix it myself and put altogether at this PR.

I really appreciate any more advices to come from you. It would be great.
(Please bear in mind that I'm still at least just 1/3 of a "job done"
milestone. So there's a lot of ghost code floating around. I'll take care
of them in the run).

I'm just making a break to get feedback from you all and to recharge my
"code energy" for the next round.

Em Dom, 14 de jun de 2015 13:48, Michal Cichra <notifications@github.com>
escreveu:

> In lib/highline/import.rb
> <https://github.com/JEG2/highline/pull/140#discussion_r32381638>:
>
> > @@ -1,3 +1,6 @@
> > +#!/usr/bin/env ruby
>
> Why the shebang? Most of the files can't be executed standalone.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/JEG2/highline/pull/140/files#r32381638>.
>